shared/src/business/entities/customCaseReportSearch/CustomCaseReportSearch.ts

```diff
@@ -35,10 +35,7 @@ export class CustomCaseReportSearch extends JoiValidationEntity {
   public pageSize: number;
```

Comment:
`CustomCaseSearch` was moved from OpenSearch to Postgres. The refactors below reflect that.